src/DynamoCoreWpf/ViewModels/Core/OutPortViewModel.cs

```diff
@@ -17,10 +17,9 @@ public partial class OutPortViewModel : PortViewModel
         private DelegateCommand hideConnectionsCommand;
         private DelegateCommand portMouseLeftButtonOnContextCommand;
 
-        private SolidColorBrush portValueMarkerColor;
+        private SolidColorBrush portValueMarkerColor = new SolidColorBrush(Color.FromArgb(255, 204, 204, 204));
 
-        private SolidColorBrush PortValueMarkerBlue;
```

Comment:
Is this PortValueMarkerBlue not being used anymore for Outports?

Reply:
Sorry, I don't know when that change even occured. I have no business in the OutPortViewModel.cs it must have been by accident. I recreated the property.